src/test/resources/vilkår/regler/UTGIFTER_OVERNATTING.json

```
@@ -14,6 +42,20 @@
           "regelId" : "SLUTT_NODE"
         }
       }
+    },
+    "HØYERE_UTGIFTER_HELSEMESSIG_ÅRSAKER" : {
+      "regelId" : "HØYERE_UTGIFTER_HELSEMESSIG_ÅRSAKER",
+      "erHovedregel" : true,
+      "svarMapping" : {
+        "JA" : {
+          "begrunnelseType" : "VALGFRI",
+          "regelId" : "SLUTT_NODE"
+        },
+        "NEI" : {
+          "begrunnelseType" : "PÅKREVD",
+          "regelId" : "SLUTT_NODE"
```

```suggestion
        "JA" : {
          "begrunnelseType" : "PÅKREVD",
          "regelId" : "SLUTT_NODE"
        },
        "NEI" : {
          "begrunnelseType" : "VALGFRI",
          "regelId" : "SLUTT_NODE"
```

Når vi gjør endringen på dette vilkåret må vi oppdatere testene også